src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -229,6 +252,21 @@ def safe_to_destroy(self, ids):
                 'is_safe_to_destroy': False,
             }
 
+    @Endpoint('GET', query_params=['svc_ids'])
+    @ReadPermission
+    @raise_if_no_orchestrator
+    @handle_orchestrator_error('osd')
+    def safe_to_delete(self, svc_ids):
```

Comment:
I'd prefer a `/api/osd/<id>/safe_to_delete` rather than `/api/osd/safe_to_delete?svd_ids=...`. But let's be pragmatic (and there already are other endpoints in the Dashboard that follow this practice) :-/